recipe/no_undefined.patch

```diff
@@ -0,0 +1,11 @@
+--- a/Makefile.am
++++ b/Makefile.am
+@@ -164,7 +164,7 @@
+ 	isl_vertices_private.h \
+ 	isl_vertices.c
+ libisl_la_LIBADD = @MP_LIBS@
+-libisl_la_LDFLAGS = -version-info @versioninfo@ \
++libisl_la_LDFLAGS = -version-info @versioninfo@ -no-undefined \
```

Comment:
Does this one need to have `--no-undefined` and not `-no-undefined`? That was the latest build error.